web/src/main/java/io/openshift/appdev/missioncontrol/web/api/websocket/MissionControlStatusEndpoint.java

```diff
@@ -0,0 +1,68 @@
+package io.openshift.appdev.missioncontrol.web.api.websocket;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import javax.enterprise.event.Observes;
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObjectBuilder;
+import javax.websocket.OnClose;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+import javax.websocket.server.PathParam;
+import javax.websocket.server.ServerEndpoint;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.openshift.appdev.missioncontrol.core.api.StatusMessage;
+import io.openshift.appdev.missioncontrol.core.api.StatusMessageEvent;
+
+/**
+ * A websocket based resource that informs clients about the status of the operations
+ *
+ * https://abhirockzz.wordpress.com/2015/02/10/integrating-cdi-and-websockets/
+ */
+@ServerEndpoint(value = "/status/{uuid}")
+public class MissionControlStatusEndpoint {
+    private static final Logger log = Logger.getLogger(MissionControlStatusEndpoint.class.getName());
+
+    private static Map<UUID, Session> peers = Collections.synchronizedMap(new HashMap<>());
```

Comment:
See http://stackoverflow.com/questions/510632/whats-the-difference-between-concurrenthashmap-and-collections-synchronizedmap